torchci/lib/bot/acceptBot.ts

```diff
@@ -57,6 +58,56 @@ function acceptBot(app: Probot): void {
       }
     }
   });
+
+  app.on("pull_request.labeled", async (ctx) => {
+    const owner = ctx.payload.repository.owner.login;
+    const repo = ctx.payload.repository.name;
+    const issue_number = ctx.payload.pull_request.number;
+    ctx.payload.pull_request;
+
+    async function isApproved(): Promise<boolean> {
+      const reviews = await ctx.octokit.pulls.listReviews({
+        owner,
+        repo,
+        pull_number: issue_number,
+      });
+      const isApproved =
+        reviews.data.filter((review) => review.state == "approved").length > 0;
+      return isApproved;
+    }
+
+    if (ctx.payload.label.name === ACCEPT_2_RUN) {
+      if (await isApproved()) {
+        await ctx.octokit.issues.addLabels({
+          owner,
+          repo,
+          issue_number,
+          labels: [CIFLOW_TRUNK_LABEL],
+        });
+        await ctx.octokit.issues.removeLabel({
+          owner,
+          repo,
+          issue_number,
+          name: ACCEPT_2_RUN,
+        });
+      }
+    } else if (ctx.payload.label.name === ACCEPT_2_SHIP) {
+      if (await isApproved()) {
+        await ctx.octokit.issues.createComment({
+          owner,
+          repo,
+          issue_number,
+          body: ACCEPT_MESSAGE,
+        });
+        await ctx.octokit.issues.removeLabel({
```

Comment:
Why remove the label?

Reply:
This is probably related to https://github.com/pytorch/test-infra/pull/544

> Once someone accepts, we remove the label to prevent it from running again

One label - one approval I guess

Reply:
Ah right, the problem was that a second PR acceptance would kick off a second set of merge reviews.

Reply:
yup, exactly!